app/views/core_induction_programmes/mentor_material_parts/show_delete.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= govuk_breadcrumbs breadcrumbs: mentor_material_breadcrumbs(current_user, @mentor_material_part.mentor_material) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h2 class="govuk-heading-l">Confirm you would like to remove <i><%= @mentor_material_part.title %></i> from <%= @mentor_material_part.mentor_material.title %>.</h2>
```

Comment:
This should be `h1`, right? Possibly in the lesson part show delete too...

Reply:
yep. probably means it needs a test adding too!